tests/e2e/env/setup.sh

```diff
@@ -310,7 +313,10 @@ echo "Creating screenshots directory"
 mkdir -p $WCP_ROOT/screenshots
 
 echo "Disabling rate limiter for card declined in E2E tests"
-cli wp option add wcpay_session_rate_limiter_disabled_wcpay_card_declined_registry yes
+cli wp option set wcpay_session_rate_limiter_disabled_wcpay_card_declined_registry yes
```

Comment:
`wp option add` had problems when the option was already there, switched it with `wp option set` which doesn't fail when the option already exists.